File: Assignment40/program3.cpp
/////////////////////////////////////////////////////////////////// 
//
// File name : program3.cpp
// Description : Pattern printing using recursion
// Author : Adinath Santosh Pawar
// Date : 21/7/25
// 
///////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

void Display()
{
    static int i = 5;

    if(i >= 1)
    {
        cout<<i<<" ";
        i--;
        Display();
    }
}


int main()
{
    Display();

    return 0;
}

File: Assignment48/program3.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program3.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 20/8/25           
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(String str)
    {
        for(int i = 0; i < str.length(); i++)
        {
            for(int j = 0; j < str.length(); j++)
            {
                if(i >= j)
                {
                    System.out.print(str.charAt(j) + " ");
                }
            }
            System.out.println();
        }
    }
}

class program3
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);

       System.out.print("Enter the string ");
       String str = sobj.nextLine();

       pobj.Pattern(str);
    }
}